sonic-net/sonic-buildimage
# [202205][Mellanox] Add mlxtrace to techsupport (#15961)

Backport of https://github.com/sonic-net/sonic-buildimage/pull/15961
#### Why I did it
Added the fwtrace config files in order to be able to call the `mlxtrace` utility during the `show techsupport` dump.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added fwtrace config files. Added path to these files to sai.profile for each mlnx device.

#### How to verify it
Execute the `show techsupport` command and check if `mlxtrace` output is in the system dump.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
